.github/workflows/_ci.yaml

```diff
@@ -612,34 +619,169 @@ jobs:
+  #       failed_tests=$(echo $summary_line | grep -oE '[0-9]+ failed' | awk '{print $1} END { if (!NR) print 0}')
+  #       passed_tests=$(echo $summary_line | grep -oE '[0-9]+ passed' | awk '{print $1} END { if (!NR) print 0}')
+  #       total_tests=$((failed_tests + passed_tests))
+  #       echo "TOTAL_TESTS=${total_tests}" >> $GITHUB_OUTPUT
+  #       echo "ERRORS=${errors}" >> $GITHUB_OUTPUT
+  #       echo "PASSED_TESTS=${passed_tests}" >> $GITHUB_OUTPUT
+  #       echo "FAILED_TESTS=${failed_tests}" >> $GITHUB_OUTPUT
+  #     ARTIFACTS: |
+  #       test-gemma.log
+  #   secrets: inherit
+
+  # test-maxtext:
+  #   needs: build-maxtext
+  #   if: inputs.ARCHITECTURE == 'amd64' # no arm64 gpu runners
+  #   uses: ./.github/workflows/_test_maxtext.yaml
+  #   with:
+  #     MAXTEXT_IMAGE: ${{ needs.build-maxtext.outputs.DOCKER_TAG_FINAL }}
+  #   secrets: inherit
+
+  test-axlearn-eks:
+    needs: build-axlearn
     if: inputs.ARCHITECTURE == 'amd64'
-    with:
-      TEST_NAME: gemma
-      EXECUTE: |
-        docker run --shm-size=1g --gpus all ${{ needs.build-gemma.outputs.DOCKER_TAG_FINAL }} \
-        bash -ec \
-        "cd /opt/gemma && pip install -e .[dev] && pytest ." | tee test-gemma.log
-      STATISTICS_SCRIPT: |
-        summary_line=$(tail -n1 test-gemma.log)
-        errors=$(echo $summary_line | grep -oE '[0-9]+ error' | awk '{print $1} END { if (!NR) print 0}')
-        failed_tests=$(echo $summary_line | grep -oE '[0-9]+ failed' | awk '{print $1} END { if (!NR) print 0}')
-        passed_tests=$(echo $summary_line | grep -oE '[0-9]+ passed' | awk '{print $1} END { if (!NR) print 0}')
+    runs-on: eks
+    env:
+      AXLEARN_DOCKER_IMAGE: ${{ needs.build-axlearn.outputs.DOCKER_TAG_FINAL }}
+      JOB_NAME: axlearn-${{ github.run_id }}
+      TOKEN_NAME: axlearn-${{ github.run_id }}-token
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v4
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: K8s GHCR login and delete
+      uses: ./.github/actions/store-delete-k8s-ghcr
+      with: 
+          token-name: ${{ env.TOKEN_NAME }}
+    - name: Configure axlearn test job
+      run: |
+        # Replace placeholders in axlearn-job.yml with environment variables
+        yq -i ea '
+           select(di == 0).metadata.name = strenv(JOB_NAME)
+          | select(di == 0).spec.template.spec.containers[0].image = strenv(AXLEARN_DOCKER_IMAGE)
+          | select(di == 0).spec.template.spec.containers[1].env[0].value = "${{ github.run_id }}"
+          | select(di == 0).spec.template.spec.imagePullSecrets[].name = strenv(TOKEN_NAME)' \
+        .github/eks-workflow-files/axlearn/axlearn-job.yml
+        git diff .github/eks-workflow-files/axlearn/axlearn-job.yml
```

Comment:
If you haven't already then format the file committed to the repository with `yq`, so the diff only shows the substituted placeholders and no other whitespace changes.